subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmComponentDependencies.java

```diff
@@ -83,6 +85,19 @@ public Dependency testFixtures(ModuleDependency moduleDependency) {
         return moduleDependency;
     }
 
+    @Inject
+    protected abstract Project getCurrentProject();
+
+    @Inject
+    protected abstract ObjectFactory getObjectFactory();
```

Comment:
These should be next to `getDependencyFactory()`, and `getDependencyFactory()` should be made `abstract` to match.